From d48430593be88a5199f6f9fd1ead833fb2eb2cc5 Mon Sep 17 00:00:00 2001
From: George Kiagiadakis <george.kiagiadakis@collabora.com>
Date: Tue, 30 Sep 2014 01:13:36 +0300
Subject: [PATCH 01/16] CMakeLists.txt: actually require GStreamer 1.2.0

---
 CMakeLists.txt | 16 ++++++++--------
 1 file changed, 8 insertions(+), 8 deletions(-)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index 574e31f..4d3e7dd 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -78,26 +78,26 @@ endif()
 set(QTGSTREAMER_INCLUDE_DIR ${CMAKE_SOURCE_DIR}/src)
 include(QtGStreamerConfigCommon)
 
-find_package(GStreamer 1.0.0 COMPONENTS base)
+find_package(GStreamer 1.2.0 COMPONENTS base)
 macro_log_feature(GSTREAMER_FOUND "GStreamer" "Required to build QtGStreamer"
-                                  "http://gstreamer.freedesktop.org/" TRUE "1.0.0")
+                                  "http://gstreamer.freedesktop.org/" TRUE "1.2.0")
 macro_log_feature(GSTREAMER_BASE_LIBRARY_FOUND "GStreamer base library"
                                                "Used for building the ${QTVIDEOSINK_NAME} element"
-                                               "http://gstreamer.freedesktop.org/" FALSE "1.0.0")
+                                               "http://gstreamer.freedesktop.org/" FALSE "1.2.0")
 
-find_package(GStreamerPluginsBase 1.0.0 COMPONENTS app audio video pbutils)
+find_package(GStreamerPluginsBase 1.2.0 COMPONENTS app audio video pbutils)
 macro_log_feature(GSTREAMER_APP_LIBRARY_FOUND "GStreamer app library"
                                               "Required to build QtGStreamerUtils"
-                                              "http://gstreamer.freedesktop.org/" TRUE "1.0.0")
+                                              "http://gstreamer.freedesktop.org/" TRUE "1.2.0")
 macro_log_feature(GSTREAMER_AUDIO_LIBRARY_FOUND "GStreamer audio library"
                                                 "Required to build QtGStreamer"
-                                                "http://gstreamer.freedesktop.org/" TRUE "1.0.0")
+                                                "http://gstreamer.freedesktop.org/" TRUE "1.2.0")
 macro_log_feature(GSTREAMER_VIDEO_LIBRARY_FOUND "GStreamer video library"
                                                 "Required to build QtGStreamer"
-                                                "http://gstreamer.freedesktop.org/" TRUE "1.0.0")
+                                                "http://gstreamer.freedesktop.org/" TRUE "1.2.0")
 macro_log_feature(GSTREAMER_PBUTILS_LIBRARY_FOUND "GStreamer pbutils library"
                                                 "Used for building the Discoverer API"
-                                                "http://gstreamer.freedesktop.org/" TRUE "1.0.0")
+                                                "http://gstreamer.freedesktop.org/" TRUE "1.2.0")
 
 find_package(GLIB2)
 macro_log_feature(GLIB2_FOUND "GLib" "Required to build QtGLib" "http://www.gtk.org/" TRUE)
-- 
2.8.3

